docs-markdown/src/controllers/image-controller.ts

```diff
@@ -385,7 +394,7 @@ export async function applyLocScope() {
         return;
     }
     // if user has not selected any text, then continue
-    const RE_LOC_SCOPE = /:::image\s+((source|type|alt-text|lightbox|border)="([a-zA-Z0-9_.\/ -]+)"\s*)+:::/gm;
+    const RE_LOC_SCOPE = /:::image\s+((source|type|alt-text|lightbox|border|link)="([a-zA-Z0-9_.\/ -]+)"\s*)+:::/gm;
```

Comment:
I dont think this will support other locales/foreign languages. I think we need `.*?`. I think the same problem exists with all these regexes actually.

Reply:
https://github.com/microsoft/vscode-docs-authoring/pull/589#discussion_r428336650